### PR TITLE
Add idiv asia collection

### PR DESCRIPTION
**Pull request by chorng**

### CODE REVIEW

**Comment by chorng** [comment]
@Dorothyrono Please review the entry of the idiv asia collection

File: collections/idiv-central-asia.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        - Name: Description
+          Title: Description
+      Rows:
+        - Name: crop
+          Description: major crop type
+        - Name: season
+          Description: season of the crop
+Contact: ml@radiant.earth
+Provider: "[Radiant Earth](https://radiant.earth/)"
+ManagedBy: "[Source Cooperative](https://beta.source.coop/repositories/idiv/asia-crop-type/description)"
```
Comment (Dorothyrono) [inline collections/idiv-central-asia.yaml:27]: sentinel Hub

File: collections/idiv-central-asia.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  - crop type
+  - machine learning
+  - open data
+License: "[CC BY 4.0](https://creativecommons.org/licenses/by/4.0/)"
+Resources:
+  - Group: Sentinel Hub Resources
+    EndPoint: https://services.sentinel-hub.com
+    Name: Sentinel Hub
+    Role: processor
+    Type: 0baca295-e267-42b1-b535-fe899259f791
```
Comment (Dorothyrono) [inline collections/idiv-central-asia.yaml:40]: `byoc-0baca295-e267-42b1-b535-fe899259f791`

File: collections/idiv-central-asia.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      Rows:
+        - Name: crop
+          Description: major crop type
+        - Name: season
+          Description: season of the crop
+Contact: ml@radiant.earth
+Provider: "[Radiant Earth](https://radiant.earth/)"
+ManagedBy: "[Source Cooperative](https://beta.source.coop/repositories/idiv/asia-crop-type/description)"
+Tags:
+  - raster
```
Comment (Dorothyrono) [inline collections/idiv-central-asia.yaml:29]: `mlhub-label`

**Review by Dorothyrono — CHANGES_REQUESTED** [review]

**Comment by Dorothyrono** [comment]
I see there is a publication that can be added

 https://app.dimensions.ai/details/publication/pub.1129684591?and_subset_figshare_doi=10.6084%2Fm9.figshare.12047478

File: collections/idiv-central-asia.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
+Name:  A crop type dataset for consistent land cover classification in Central Asia
+Description: |
+  Land cover is a key variable in the context of climate change. In particular, crop type information is essential to understand the spatial distribution of water usage and anticipate the risk of water scarcity and the consequent danger of food insecurity. This applies to arid regions such as the Aral Sea Basin (ASB), Central Asia, where agriculture relies heavily on irrigation. Here, remote sensing is valuable to map crop types, but its quality depends on consistent ground-truth data. Yet, in the ASB, such data is missing. Addressing this issue, we collected thousands of polygons on crop types, 97.7% of which in Uzbekistan and the remaining in Tajikistan. We collected 8,196 samples between 2015 and 2018, 213 in 2011 and 26 in 2008. Our data compiles samples for 40 crop types and is dominated by Ã¢Â€ÂœcottonÃ¢Â€Â (40%) and Ã¢Â€ÂœwheatÃ¢Â€Â, (25%). These data were meticulously validated using expert knowledge and remote sensing data and relied on transferable, open-source workflows that will assure the consistency of future sampling campaigns.
```
Comment (Dorothyrono) [inline collections/idiv-central-asia.yaml]: The description should be modified, as it is exactly similar to what is in the source cooperative page. 

Here is my suggested description

`This dataset collection provides crop type dataset for consistent land cover classification in Central Asia. 8,196 samples were collected between 2015 and 2018, 213 in 2011 and 26 in 2008. The  data compiles samples for 40 crop types and is dominated by cotton (40%) and wheat, (25%). The data went through validation process using expert knowledge and remote sensing data and relied on transferable, open-source workflows that will assure the consistency of future sampling campaigns. This dataset (crop type information) is essential to understand the spatial distribution of water usage and anticipate the risk of water scarcity and the consequent danger of food insecurity especially to arid regions such as the Aral Sea Basin (ASB), Central Asia, where agriculture relies heavily on irrigation. More information can be found in the documentation page [[Source Cooperative](https://beta.source.coop/repositories/idiv/asia-crop-type/description)](https://beta.source.coop/repositories/idiv/asia-crop-type/description)`

File: collections/idiv-central-asia.yaml [hunk 5]
```diff
@@ -0,0 +1,59 @@
+Name:  A crop type dataset for consistent land cover classification in Central Asia
```
Comment (Dorothyrono) [inline collections/idiv-central-asia.yaml:1]: I think the name is too long ?

File: collections/idiv-central-asia/README.MD [hunk 6]
```diff
@@ -0,0 +1,296 @@
+# A crop type dataset for consistent land cover classification in Central Asia
+
+## Short description
+
+Land cover is a key variable in the context of climate change. In particular, crop type information is essential to understand the spatial distribution of water usage and anticipate the risk of water scarcity and the consequent danger of food insecurity. This applies to arid regions such as the Aral Sea Basin (ASB), Central Asia, where agriculture relies heavily on irrigation. Here, remote sensing is valuable to map crop types, but its quality depends on consistent ground-truth data. Yet, in the ASB, such data is missing. Addressing this issue, we collected thousands of polygons on crop types, 97.7% of which in Uzbekistan and the remaining in Tajikistan. We collected 8,196 samples between 2015 and 2018, 213 in 2011 and 26 in 2008. Our data compiles samples for 40 crop types and is dominated by cotton (40%) and wheat, (25%). These data were meticulously validated using expert knowledge and remote sensing data and relied on transferable, open-source workflows that will assure the consistency of future sampling campaigns.
```
Comment (Dorothyrono) [inline collections/idiv-central-asia/README.MD:5]: The description should be modified, as it is exactly similar to what is in the source cooperative page.

Here is my suggested description
`
This dataset collection provides crop type dataset for consistent land cover classification in Central Asia. 8,196 samples were collected between 2015 and 2018, 213 in 2011 and 26 in 2008. The  data compiles samples for 40 crop types and is dominated by cotton (40%) and wheat, (25%). The data went through validation process using expert knowledge and remote sensing data and relied on transferable, open-source workflows that will assure the consistency of future sampling campaigns. This dataset (crop type information) is essential to understand the spatial distribution of water usage and anticipate the risk of water scarcity and the consequent danger of food insecurity especially to arid regions such as the Aral Sea Basin (ASB), Central Asia, where agriculture relies heavily on irrigation. More information can be found in the documentation page [[Source Cooperative](https://beta.source.coop/repositories/idiv/asia-crop-type/description)](https://beta.source.coop/repositories/idiv/asia-crop-type/description)`

File: collections/idiv-central-asia/README.MD [hunk 7]
```diff
@@ -0,0 +1,296 @@
+# A crop type dataset for consistent land cover classification in Central Asia
```
Comment (Dorothyrono) [inline collections/idiv-central-asia/README.MD:1]: I think the name is too long, can it be shortened ?

**Review by Dorothyrono — CHANGES_REQUESTED** [review]

**Review by Dorothyrono — APPROVED** [review]

**Pull request merged**